### DISCUSSION TITLE
Monitor GraphQL Query

### DISCUSSION THREAD

**Discussion by jrshah12690** [apollographql/apollo-server#5567] [Q&A]
We are using React, NodeJs, GraphQL with apollo server and application insight for monitoring tool but we are getting post/graphql for all request in application insight.

So please let us know how can we monitor particular Graph API request in application insight and let us know if you know any other tool to achieve this.

Thanks in advanced.
@abernix @benjamn @jglovier

**Comment by glasser** [comment]
One option is Apollo Studio! https://www.apollographql.com/studio/
